src/cephadm/cephadm

```diff
@@ -45,7 +45,7 @@ from pathlib import Path
 FuncT = TypeVar('FuncT', bound=Callable)
 
 # Default container images -----------------------------------------------------
-DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:master'
+DEFAULT_IMAGE = 'docker.io/rhcsdashboard/ceph-exporter'
```

Comment:
i guess this is just to be used while we are developing the feature.

Reply:
Yes..the package will be be the part of the default ceph image once merged in master.